client/src/telemetry/SentryClientTelemetry.ts

```diff
@@ -18,33 +20,57 @@ export class SentryClientTelemetry implements Telemetry {
   public init(extensionState: ExtensionState) {
     this.extensionState = extensionState;
 
-    Sentry.init({
+    const integrations = getDefaultIntegrations({}).filter(
+      (defaultIntegration) => {
+        return !["BrowserApiErrors", "Breadcrumbs", "GlobalHandlers"].includes(
+          defaultIntegration.name
+        );
+      }
+    );
+
+    const client = new Sentry.NodeClient({
       dsn: this.dsn,
-      tracesSampleRate: this.extensionState.env === "development" ? 1 : 0.001,
```

Comment:
We don't need rate sampling because we don't use spans (or other sampling) techniques on the client-side?

Reply:
That's correct